### DISCUSSION TITLE
Excel custom functions with enum parameters

### DISCUSSION THREAD

**Discussion by Edward-Zhou** [OfficeDev/office-js#4651] [Q&A]
I want to implement a excel custom functions with enum parameters like XLOOKUP
![image](https://github.com/OfficeDev/office-js/assets/16079555/b68e179c-eac2-4412-9643-8166d166c5e5)
How to achieve the `match_mode` list in excel custom functions?

**Comment by dazzla5881** [comment]
hi, did you ever find an answer to this?
